assets/js/phoenix_live_view.js

```diff
@@ -511,11 +511,28 @@ export class LiveSocket {
     for(let type of ["change", "input"]){
       this.on(type, e => {
         let input = e.target
-        let key = input.type === "checkbox" ? "checked" : "value"
-        if(this.prevInput === input && this.prevValue === input[key]){ return }
+
+        let value
+        if (input.type === "checkbox") {
+          value = input["checked"]
```

Comment:
Have you found some issue using `input.value`? Because testing here I haven't 🤔 

If we use `input.value` it works when the input is a checkbox as well, so we don't need this check.

```JS
if (input.type === "select-multiple") {
  let selectedOptions = input.selectedOptions
  let values = []
  for (let i = 0; i < selectedOptions.length; i++) {
    values.push(selectedOptions.item(i).value)
  }
  // Convert to a string for easier array comparison
  value = JSON.stringify(values)
}
else {
  value = input.value
}
```

Reply:
Nevermind, here we want to get the information if whether the input is `checked` or not. If we use `input.value` for checkbox we get different values:

```JS
checkbox.checked
=> false

checkbox.value
=> "true"
```